test/modules/encoders/test_albef_text_encoder.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from test.test_utils import assert_expected, set_rng_seed
+from torch import Tensor
+from torchmultimodal.modules.encoders.albef_text_encoder import ALBEFTextEncoder
+
+
+class TestALBEFTextEncoder:
+    set_rng_seed(0)
+    text_encoder = ALBEFTextEncoder(hidden_size=3, num_attention_heads=1)
```

Comment:
The reason I had `set_rng_seed` in every method is because if I only set seed once in `init`, the outputs are different when I run `pytest` on a single file and run `pytest` on all test files. I just tried fixture and it seems to be the same case.

Reply:
The fixture should be called/refreshed by each test method. 

How about this:
```
@pytest.fixture(autouse=True)
def some_fixture(self):
    set_rng_seed(40)
```
By setting `autouse=True` the fixture will be automatically called in each of the test method.

Reply:
@katrina433 can you follow @langong347's suggestion here?